recipes/omsl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "omsl" %}
+{% set mslversion = "3.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ mslversion }}
+
+source:
+  url: https://github.com/OpenModelica/OpenModelica-ModelicaStandardLibrary/archive/e68d7a0317c565ee00c1b8d44c527979ea0304bf.zip
+  sha256: 16dbb64fc3e2badde01cecd6af73b95f88e12355e393abeb827927beba3fc289
+
+build:
+  script_env:
+   - MSLVERSION={{ mslversion }}
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  host:
+    - pip
+  run:
+    - omcompiler
```

Comment:
```suggestion
```
omsl is a library right? It should not depend on a compiler package at runtime. Users should know that they need a compiler to use these files. It seems that this library is very similar to a C/CXX header-only library i.e. Eigen. The Eigen package is a collection of source files, and it does not depend on a compiler.